pkgs/by-name/db/dbd/package.nix

```diff
@@ -0,0 +1,32 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+}:
+
+stdenv.mkDerivation {
+  pname = "dbd";
+  version = "1.50-unstable-2016-01-04";
+
+  src = fetchFromGitHub {
+    owner = "gitdurandal";
+    repo = "dbd";
+    rev = "8cf5350781b6753fcdd863148a5dcc6976e693ca";
+    hash = "sha256-b2yBZ2/Ab+SviKNlyZgdfiZ7GGZ1sonZnblD0i+vuFw=";
+  };
+
+  makeFlags = [
+    "PREFIX=${placeholder "out"}/local"
+    "CC=cc"
+  ];
+
+  buildFlags = lib.optional stdenv.hostPlatform.isDarwin [ "darwin" ] ++ lib.optional stdenv.hostPlatform.isUnix [ "unix" ] ++ lib.optional stdenv.hostPlatform.isWindows [ "win32" ];
```

Comment:
I guess this will evaluate to `[ [ "darwin" ] [ "unix" ] ]` on darwin?

Reply:
In theory, according to the [Makefile](https://github.com/gitdurandal/dbd/blob/master/Makefile), `unix` should evaluate only `Linux, NetBSD, FreeBSD, OpenBSD`. Maybe the developer just mislabeled.

Reply:
I will remove windows building because failing.